app/src/main/java/ch/ubique/notifyme/app/utils/StringUtils.java

```diff
@@ -30,9 +30,26 @@ public static SpannableString getTwoColoredString(String wholeString, String sub
 		return spannable;
 	}
 
+	/**
+	 * Formats a duration in milliseconds to a String of hours, minutes and seconds, or to only hours and minutes if the
+	 * duration is more than 10 hours
+	 * @param duration in milliseconds
+	 * @return a formatted duration String
+	 */
+	public static String getShortDurationString(long duration) {
+		if (duration >= 10 * ONE_HOUR) {
```

Comment:
Since we're already using TimeUnit to convert milliseconds to other units, the ONE_HOUR constant could also use that method for consistency